app/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           xmlns:tools="http://schemas.android.com/tools"
           android:versionCode="106"
-          android:versionName="2.56">
+          android:versionName="2.55.0">
```

Comment:
this should remain unchanged.

Reply:
yeah, I wonder if I'm missing something as this is updated by the mobile script. Let me fix it

Reply:
Done